docs/content/en/docs/concepts/architecture/components/KLTcomponents.md

```diff
@@ -0,0 +1,34 @@
+---
+title: Lifecycle Toolkit Components
+linktitle: KLT Components
+description: Basic understanding of Keptn Lifecycle Toolkit Components
+weight: 80
+cascade:
+---
+
+### Keptn Lifecycle Toolkit Components
+
+The Keptn Lifecycle Toolkit consists of two main components:
+
+* Keptn Lifecycle Operator, splits into two separate operators:
+  * Lifecycle-Operator
+  * Metrics-Operator
+* Keptn Lifecycle Scheduler
+
+```mermaid
+graph TD;
+    KLTComponents-->Operators;
+    KLTComponents-->Scheduler
+   Operators-->Lifecycle-Operator
+   Operators-->Metrics-Operator
+
+style KLTComponents fill:#006bb8,stroke:#fff,stroke-width:px,color:#fff
+style Operators fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+style Scheduler fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+style Lifecycle-Operator fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+style Metrics-Operator fill:#d8e6f4,stroke:#fff,stroke-width:px,color:#006bb8
+
+```
+**Keptn's Lifecycle Operator** is a Kubernetes operator that automates the deployment and management of Keptn's Lifecycle Controller components in a Kubernetes cluster.
```

Comment:
```suggestion
**Keptn's Lifecycle Operator** is
a Kubernetes [operator](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/)
that automates the deployment and management
of Keptn's Lifecycle Controller components in a Kubernetes cluster.
```
The markdown-linter is going to fail lines that are longer than 145 characters, I think -- I'm not sure why it wasn't run as part of the CI for this PR but it will get you eventually so let's shorten the line.

And let's link "operator" to some k8s docs.  Yeah, people should know this stuff but...  Just a link is not intrusive.

Reply:
Oh yes,I forgot about the markdown for this.I need to check.